src/domains/Invite/SelectFile/utils.ts

```diff
@@ -0,0 +1,19 @@
+import Papa from 'papaparse';
+
+const defaultConfig = {
+    header: true,
+    skipEmptyLines: true,
+    complete: (results: Array<string>) => {
+        console.log(results);
+    },
+};
+
+function csv(file: File, config = {}) {
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    const merged: object = { ...defaultConfig, ...config };
```

Comment:
Just for the highest amount of type checking possible, we can probably import a type like `ConfigOptions` from papaparse too.  `import Papa, { ConfigOptions } from 'papaparse'`.  I'm not sure if the name is actually ConfigOptions, but I think you get the idea.

The reason this is a good idea is that if someone absent-mindedly or accidentally updates papaparse to a version where the config is now different (lets say they rename `header` to `headers` for whatever reason), then typescript will catch this for us.